.gitignore

```diff
@@ -78,3 +78,5 @@ android/build/
 samples/android/.externalNativeBuild/
 samples/android/.gradle/
 samples/android/build/
+tests/union_vector/union_vector_generated.js
```

Comment:
Generated code should be part of the commit. We do that such that we can spot the effect of future changes to the code  generators.